Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Ride.hs

```diff
@@ -44,7 +44,12 @@ import qualified Tools.Maps as MapSearch
 import Tools.Metrics
 import qualified Tools.Notifications as Notify
 
-type GetDriverLocResp = MapSearch.LatLong
+data GetDriverLocResp = GetDriverLocResp
+  { lat :: Double,
+    lon :: Double,
+    lastUpdate :: Maybe UTCTime
```

Comment:
It can be mandatory, because you always put `Just`

Reply:
I will send the maybe type then, frontend guys will handle it accordingly

Reply:
you can send the value without maybe, it's not an issue.